tests/integration/CreateSprintTemplate.test.ts

```diff
@@ -57,81 +59,131 @@ describe("Login Page:", () => {
     console.log("clicking sprint template");
     await page.waitForSelector("#template-name");
     await page.type("#template-name", randomTemplateName);
-    const origin1 = await page.waitForSelector(`[id="Talk to Customers"]`);
-    const origin2 = await page.waitForSelector(`[id="No Nicotine or Tobacco"]`);
-    const origin3 = await page.waitForSelector(`[id="3 Hours of Deep Work"]`);
-    const morning = await page.waitForSelector("#Morning");
-    const workDay = await page.waitForSelector("#Workday");
-    const evening = await page.waitForSelector("#Evening");
-    let ob = await origin1?.boundingBox();
-    // let ob2 = await origin2?.boundingBox(); // <-----start ||Currently not using this box as unless you make some sort of timer function, it will fail as it will try to drag the box before it has moved to the correct location.
-    // let ob3 = await origin3?.boundingBox(); // <--------end
-    let mb = await morning?.boundingBox();
-    let wb = await workDay?.boundingBox();
-    let eb = await evening?.boundingBox();
+
+    // Drag and drop
+    const origin1 = (await page.waitForSelector(
+      `[id="Talk to Customers"]`
+    )) as ElementHandle<Element>;
```

Comment:
Looks a lot cleaner with the type casting in here (and the DeepSource ignore comments & !?'s removed!